scm_repo_link.go

```diff
@@ -165,6 +181,19 @@ func (c *Client) SCMRepoLinkDelete(ctx context.Context, app string) error {
 	return nil
 }
 
+func (c *Client) SCMRepoLinkPullRequest(ctx context.Context, app string, number int) (*RepoLinkPullRequest, error) {
+	var res SCMRepoLinkPullRequestResponse
+	err := c.ScalingoAPI().DoRequest(ctx, &http.APIRequest{
+		Method:   "GET",
+		Endpoint: "/apps/" + app + "/scm_repo_link/pulls/" + strconv.Itoa(number),
```

Comment:
nitpick: I find the following more readable 
```suggestion
		Endpoint: fmt.Sprintf("/apps/%s/scm_repo_link/pulls/%d", app, number),
```